cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml

```diff
@@ -0,0 +1,62 @@
+doctrine:
```

Comment:
What if the user has already configured doctrine? (which would most likely be the case). 

It is good practice not touch other bundles configuration. Please remove `doctrine`, `framework` and `security`.

Reply:
This config just extend base config, not replace it. I know, It looks not good, but another way you should write more settings manually.

Reply:
Yeah, I would prefer you add a post-install comment and explain the manual steps.